packages/storefront-lwc/src/modules/commerce/basket/basket.html

```diff
@@ -62,9 +62,9 @@ <h1 class="page-title">Your basket</h1>
                     </a>
                 </div>
                 <div class="col-sm-3 text-center">
-                    <h2 class="number-of-items">
+                    <p class="number-of-items">
                         {basket.totalProductsQuantity} Items
```

Comment:
This is inside a container. We shouldnt need to have multiple containers on one page. Are we going to clean up this implementation within this story or do we have a separate story?

Reply:
There are also several un needed classes and divs from a copy-paste.

Reply:
done